packages/trader/src/Modules/Contract/Containers/replay-chart.tsx

```diff
@@ -115,7 +115,7 @@ const ReplayChart = observer(
                 isConnectionOpened={is_socket_opened}
                 isStaticChart={
                     // forcing chart reload when start_epoch changes to an earlier epoch for ACCU closed contract:
-                    is_accumulator_contract && end_epoch && Number(start_epoch) < Number(prev_start_epoch)
+                    !!is_accumulator_contract && !!end_epoch && Number(start_epoch) < Number(prev_start_epoch)
```

Comment:
I think `is_accumulator_contract` is already a boolean 🤔

Reply:
@balakrishna-deriv unfortunately, it can be undefined when it's not passed in props (it's an optional prop ) :)

Reply:
It can be `undefined`, but I still don't understand why we need `!!` as type will be converted to falsy/truthy anyway. Same for `end_epoch`